### PR TITLE
update makefile targets 

### PR DESCRIPTION
**Pull request by ranakan19**
fix wrt: https://issues.redhat.com/browse/CRT-904

closing the previous https://github.com/codeready-toolchain/toolchain-e2e/pull/286 due to PR name to comply with PR pairing logic.

### CODE REVIEW

**Review by xcoulon — APPROVED** [review]

File: make/test.mk [hunk 1]
```diff
@@ -355,9 +309,8 @@ deploy-operator:
...
 	$(eval MEMBER_OPERATOR_WEBHOOK_IMAGE := $(shell cat ${IMAGE_NAMES_DIR}/member-operator-webhook))
 	$(eval COMPONENT_IMAGE_REPLACEMENT := ${COMPONENT_IMAGE_REPLACEMENT};s|REPLACE_MEMBER_OPERATOR_WEBHOOK_IMAGE|${MEMBER_OPERATOR_WEBHOOK_IMAGE}|g)
-ifeq ($(IS_OS_3),)
-		# it is not using OS 3 so we will install operator via CSV
-		$(eval NAME_SUFFIX := ${QUAY_NAMESPACE}-${RESOURCES_SUFFIX})
+	# install operator via CSV
+	$(eval NAME_SUFFIX := ${QUAY_NAMESPACE}-${RESOURCES_SUFFIX})
     ifeq ($(ENV_YAML),)
 		# ENV_YAML is not set
 		$(eval ENV_YAML := ${E2E_REPO_PATH}/deploy/env/${ENVIRONMENT}.yaml)
```
Comment (MatousJobanek) [inline make/test.mk:316]: it's just a detail but the whole part on line 314-343 should have one indentation less

**Review by MatousJobanek — APPROVED** [review]
Nice :+1:

File: make/test.mk [hunk 2]
```diff
@@ -305,21 +267,13 @@ ifeq ($(SET_IMAGE_NAME),)
...
-				$(eval IMAGE_NAME := docker.io/${GO_PACKAGE_ORG_NAME}/${REPO_NAME}:${DATE_SUFFIX})
-				$(MAKE) -C ${E2E_REPO_PATH} build
-				docker build -f ${E2E_REPO_PATH}/build/Dockerfile -t ${IMAGE_NAME} ${E2E_REPO_PATH}
-            else
-            	# if is using OS4 then use quay registry
+            	#if is running locally, then build the image and push to quay repository
 				$(eval IMAGE_NAME := quay.io/${QUAY_NAMESPACE}/${REPO_NAME}:${DATE_SUFFIX})
 				$(MAKE) -C ${E2E_REPO_PATH} docker-push QUAY_NAMESPACE=${QUAY_NAMESPACE} IMAGE_TAG=${DATE_SUFFIX}
 				curl https://quay.io/api/v1/repository/${QUAY_NAMESPACE}/${REPO_NAME} 2>/dev/null | jq -r '.tags."${DATE_SUFFIX}".manifest_digest' > ${IMAGE_NAMES_DIR}/${REPO_NAME}_digest
 				if [[ ${REPO_NAME} == "member-operator" ]]; then \
```
Comment (MatousJobanek) [inline make/test.mk:274]: this one could have one indentation less too

File: make/test.mk (hunk 1 above)
Comment (ranakan19) [reply make/test.mk:316]: hmm, the indentation looks fine to me when i open it in local.

Comment (MatousJobanek) [reply make/test.mk:316]: you removed the lines
```
ifeq ($(IS_OS_3),)
else
   ...
endif
```
but everything what was inside this if-else-endif was indented to reflect that it was inside the if statement. After the removal of the if statement we should remove the additional indentation - like this:
```
.PHONY: deploy-operator
deploy-operator:
	$(eval IMAGE_NAME := $(shell cat ${IMAGE_NAMES_DIR}/${REPO_NAME}))
	@echo Using image ${IMAGE_NAME} and namespace ${NAMESPACE} for the repository ${REPO_NAME}
	$(eval REGISTRATION_SERVICE_IMAGE_NAME := $(shell cat ${IMAGE_NAMES_DIR}/registration-service))
	$(eval COMPONENT_IMAGE_REPLACEMENT := ;s|REPLACE_REGISTRATION_SERVICE_IMAGE|${REGISTRATION_SERVICE_IMAGE_NAME}|g)
	$(eval MEMBER_OPERATOR_WEBHOOK_IMAGE := $(shell cat ${IMAGE_NAMES_DIR}/member-operator-webhook))
	$(eval COMPONENT_IMAGE_REPLACEMENT := ${COMPONENT_IMAGE_REPLACEMENT};s|REPLACE_MEMBER_OPERATOR_WEBHOOK_IMAGE|${MEMBER_OPERATOR_WEBHOOK_IMAGE}|g)
	# install operator via CSV
	$(eval NAME_SUFFIX := ${QUAY_NAMESPACE}-${RESOURCES_SUFFIX})
ifeq ($(ENV_YAML),)
	# ENV_YAML is not set
	$(eval ENV_YAML := ${E2E_REPO_PATH}/deploy/env/${ENVIRONMENT}.yaml)
endif
	curl -sSL https://raw.githubusercontent.com/codeready-toolchain/api/master/scripts/enrich-by-envs-from-yaml.sh | bash -s -- ${E2E_REPO_PATH}/hack/deploy_csv.yaml ${ENV_YAML} > /tmp/${REPO_NAME}_deploy_csv_${DATE_SUFFIX}_source.yaml
	sed -e 's|REPLACE_IMAGE|${IMAGE_NAME}|g;s|^  name: .*|&-${NAME_SUFFIX}|;s|^  configMap: .*|&-${NAME_SUFFIX}|${COMPONENT_IMAGE_REPLACEMENT}' /tmp/${REPO_NAME}_deploy_csv_${DATE_SUFFIX}_source.yaml > /tmp/${REPO_NAME}_deploy_csv_${DATE_SUFFIX}.yaml
	cat /tmp/${REPO_NAME}_deploy_csv_${DATE_SUFFIX}.yaml | oc apply -f -
	# if the namespace already contains the CSV then update it
	if [[ -n `oc get csv 2>/dev/null || true | grep 'toolchain-${REPO_NAME}'` ]]; then \
		oc get cm `grep "^  name: cm" /tmp/${REPO_NAME}_deploy_csv_${DATE_SUFFIX}.yaml | awk '{print $$2}'` -n openshift-marketplace --template '{{.data.clusterServiceVersions}}' | sed 's/^. //g;s/namespace: placeholder/namespace: ${NAMESPACE}/'  | oc apply -f -; \
	fi
	sed -e 's|REPLACE_NAMESPACE|${NAMESPACE}|g;s|^  source: .*|&-${NAME_SUFFIX}|' ${E2E_REPO_PATH}/hack/install_operator.yaml > /tmp/${REPO_NAME}_install_operator_${DATE_SUFFIX}.yaml
	cat /tmp/${REPO_NAME}_install_operator_${DATE_SUFFIX}.yaml | oc apply -f -
	while [[ -z `oc get sa ${REPO_NAME} -n ${NAMESPACE} 2>/dev/null` ]] || [[ -z `oc get ClusterRoles | grep "^toolchain-${REPO_NAME}\.v"` ]] || [[ -z `oc get Roles -n ${NAMESPACE} | grep "^toolchain-${REPO_NAME}\.v"` ]]; do \
		if [[ $${NEXT_WAIT_TIME} -eq 300 ]]; then \
		   CATALOGSOURCE_NAME=`oc get catalogsource --output=name -n openshift-marketplace | grep "source-toolchain-.*${NAME_SUFFIX}"`; \
		   SUBSCRIPTION_NAME=`oc get subscription --output=name -n ${NAMESPACE} | grep "subscription-toolchain"`; \
		   echo "reached timeout of waiting for ServiceAccount ${REPO_NAME} to be available in namespace ${NAMESPACE} - see following info for debugging:"; \
		   echo "================================ CatalogSource =================================="; \
		   oc get $${CATALOGSOURCE_NAME} -n openshift-marketplace -o yaml; \
		   echo "================================ CatalogSource Pod Logs =================================="; \
		   oc logs `oc get pods -l "olm.catalogSource=$${CATALOGSOURCE_NAME#*/}" -n openshift-marketplace -o name` -n openshift-marketplace; \
		   echo "================================ Subscription =================================="; \
		   oc get $${SUBSCRIPTION_NAME} -n ${NAMESPACE} -o yaml; \
		   $(MAKE) print-operator-logs REPO_NAME=${REPO_NAME} NAMESPACE=${NAMESPACE}; \
		   exit 1; \
		fi; \
		echo "$$(( NEXT_WAIT_TIME++ )). attempt of waiting for ServiceAccount ${REPO_NAME} in namespace ${NAMESPACE}"; \
		sleep 1; \
	done
```
But this is just a detail and nothing critical :wink:

**Pull request merged**